src/components/AvatarWithImagePicker.js

```diff
@@ -54,9 +52,6 @@ const propTypes = {
         left: PropTypes.number,
     }).isRequired,
 
-    /** Flag to see if image is being uploaded */
-    isUploading: PropTypes.bool,
```

Comment:
If we remove `isUploading` prop, we also shouldn't pass it to `AvatarWithImagePicker` from `WorkspaceSettingsPage`